src/internal/m365/service/sharepoint/export.go

```diff
@@ -47,6 +47,7 @@ func ProduceExportCollections(
 		}
 
 		baseDir := path.Builder{}.
+			Append("Libraries").
```

Comment:
Can we use the path category string here instead?

Reply:
`path.LibrariesCategory` works out to lower case libraries. I personally feel like `Libraries` is better than `libraries` for the folder name. That said, we can maybe create another constant. Let me know if you have any thoughts. I'm merging it for now as there will be no logical changes.